fog/ledger/enclave/api/src/lib.rs

```diff
@@ -1,26 +1,30 @@
 // Copyright (c) 2018-2021 The MobileCoin Foundation
 
-//! APIs for MobileCoin Ledger Service Enclaves
+//! APIs for MobileCoin Ledger Service Enclave
 
 #![no_std]
+#![feature(allocator_api)]
```

Comment:
This probably isn't needed and should be removed

Reply:
Done

Reply:
done